### PR TITLE
feat: macOS CI Builds

### PR DESCRIPTION
**Pull request by sammcj**
- macOS CI builds working
- package name in tauri config

### CODE REVIEW

**Comment by sammcj** [comment]
Builds expected to fail on publishing package release until merged to main once.

Linting / formatting warnings are legit but not new

File: src-tauri/tauri.conf.json
Comment (Dax911) [inline src-tauri/tauri.conf.json:1]: Can we confirm that com.rusty.ollama is the correct format? Are there naming conventions here? obv we have com. but is .dev required? What is the accepted convention for the identifier?

**Review by Dax911 — APPROVED** [review]

File: src-tauri/tauri.conf.json
Comment (sammcj) [reply src-tauri/tauri.conf.json:1]: I just assumed it was the standard bundle naming format, which is the reverse dns of an application company/authors domain name - or com.author.applicationname but I'll check the rust docs to see if it's something special for rust.

At the end of the day surely we don't want to keep calling this thing rusty-ollama do we? This was just to un-break builds as rust does not allow you to do CI builds with the default bundle name.

Comment (sammcj) [reply src-tauri/tauri.conf.json:1]: Ah yeah confirmed for the Tauri's app generation - it's the standard bundle naming convention of domain or at least unique identifyer - https://tauri.app/v1/api/config/#bundleconfig.identifier

Comment (sammcj) [reply src-tauri/tauri.conf.json:1]: For the rust naming it seems the preference is to use _ ("snake casing") rather than -.

**Pull request merged**